src/components/KonnectorTile.jsx

```diff
@@ -25,8 +25,13 @@ const KonnectorTile = ({
   route,
   t
 }) => {
+  const validCategories = require('../config/categories')
+
   const categories = konnector.categories
-    ? konnector.categories.map(c => t(`category.${c}`))
+    ? konnector.categories
+        .map(c => (validCategories.includes(c) ? c : 'other'))
```

Comment:
List are not the best data structure to check for inclusion (as it is in O(n)). Here it is not very important since it's very very small but since it is in the render method, I think it should be dealth with. A `Set` would be better.

A Set could also be used instead of the `acc` list below.